.gitignore

```diff
@@ -17,6 +17,7 @@ build
 node_modules
 npm-debug.log
 sftp-config.json
+.idea
```

Comment:
this should be reverted; individuals' IDEs' clutter shouldn't pollute upstream repos. You can add this to your global gitignore tho!

Reply:
Ok fine. I just saw some other editors in there. will revert.